src/cogent3/app/data_store_new.py

```diff
@@ -494,6 +494,7 @@ def write(self, *, unique_id: str, data: str) -> DataMember:
         return member
 
     def write_not_completed(self, *, unique_id: str, data: str) -> DataMember:
+        (self.source / _NOT_COMPLETED_TABLE).mkdir(parents=True, exist_ok=True)
```

Comment:
@GavinHuttley 
I added this line, because in `drop_not_completed` we remove `not_completed` directory.